src/fiatconnect/saga.ts

```diff
@@ -427,18 +439,90 @@ export function* _getQuotes({
   return quotes
 }
 
+/**
+ * Given a Fiat Account to use and a list of possible quotes to select, attempts to find a quote
+ * for which the given Fiat Account is allowed to be used. Returns the first match, or null if
+ * none is found.
+ **/
+export function _selectQuoteFromFiatAccount({
+  normalizedQuotes,
+  fiatAccount,
+}: {
+  normalizedQuotes: FiatConnectQuote[]
+  fiatAccount: FiatAccount
+}) {
+  for (const normalizedQuote of normalizedQuotes) {
+    if (
+      normalizedQuote.getFiatAccountType() === fiatAccount.fiatAccountType &&
+      normalizedQuote.getFiatAccountSchema() === fiatAccount.fiatAccountSchema
```

Comment:
Probably an edge case, but if a quote returns multiple fiat account schemas for the same fiat account type, would we run into issues if their order is inconsistent across multiple quotes? We may have a saved fiat account with schema that is no longer the first in the new quote right?

Reply:
Assuming we have a single fiat account on file for the provider, the ordering of quotes doesn't matter, since there will only ever be a single quote for each account type/schema pair. If we have multiple fiat accounts on file, it's possible that a different quote/fiat account pair will be selected if the ordering of the quotes is not guaranteed.